include/jets/w.h

```diff
@@ -156,6 +156,15 @@
 
     u3_noun u3we_hmac(u3_noun);
 
+    u3_noun u3we_secp(u3_noun);
+    u3_noun u3we_comp(u3_noun);
+    u3_noun u3we_seri(u3_noun);
+    u3_noun u3we_deco(u3_noun);
+    u3_noun u3we_priv(u3_noun);
+    u3_noun u3we_make(u3_noun);
+    u3_noun u3we_sign(u3_noun);
+    u3_noun u3we_reco(u3_noun);
```

Comment:
There's a bunch here that don't show up anywhere else in the diff. There's 8 functions here, the arvo PR has 7 functions jetted, and there's only 3 `u3qe` functions implemented. Remnants of experimentation, or am I missing something?

Reply:
surplus removed